wesnoth-thot.po

```diff
@@ -1645,21 +1623,20 @@ msgstr ""
 #. [message]: speaker=Aiglondur
 #: data/campaigns/The_Hammer_of_Thursagan/scenarios/08_The_Court_of_Karrag.cfg:190
 msgid "Angarthing... I have a bad feeling about this."
-msgstr "Angarthingu... Mám velmi špatné tušení."
+msgstr "Angarthingu... Mám velmi zlé tušení."
 
 #. [message]: speaker=Angarthing
 #: data/campaigns/The_Hammer_of_Thursagan/scenarios/08_The_Court_of_Karrag.cfg:195
 msgid "Indeed. There is something very wrong here."
-msgstr "Souhlasím. Je tu něco hodně špatně."
+msgstr "Pravdu díš. Je tu něco hodně špatně."
 
-# říká Karrag (trpaslík)
 #. [message]: speaker=Karrag
 #: data/campaigns/The_Hammer_of_Thursagan/scenarios/08_The_Court_of_Karrag.cfg:200
 msgid "Hail, fellow dwarves. All Kal Kartha thanks you for your timely rescue."
 msgstr ""
-"Buďte pozdraveni, přátelé. My všichni vám děkujeme za včasnou záchranu."
+"Buďte pozdraveni, přátelé. My všichni vám za včasnou záchranu své díky "
+"skládáme."
```

Comment:
použití slovesa skládáme a jeho odtržení od zájmena vás mi přijde už dost vzletné
z mého pohledu sloveso na konci už moc trčí (není to písnička)
```suggestion
"Buďte pozdraveni, přátelé. My všichni vám za včasnou záchranu skládáme "
"své díky."
```

Reply:
Je to takové úvodní dosti obřadní oslovení, takže by to tu vyšší vzletnost uneslo. Ale budiž.